### PR TITLE
Stripe paymentelement migration

### PR DESCRIPTION
**Pull request by ultimate-tester**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
Currently Stripe is only utilized for card payments and nothing more.

Issue Number: [344](https://github.com/evershopcommerce/evershop/issues/344)

## What is the new behavior?
This PR expands it to the full range of payment methods that Stripe offers (e.g. iDeal, WeChat, Google Pay, Apple Pay, etc.). It does this by replacing the card element by a payment element where user can select their preferred payment method.

## Does this PR introduce a breaking change?
- [] Yes
- [x] No

## Other information
This is a tested but maybe not final version of the Payment Element migration

### CODE REVIEW

File: packages/evershop/src/modules/stripe/pages/frontStore/checkout/Stripe.jsx [hunk 1]
```diff
@@ -18,10 +18,19 @@ const stripeLoader = (publishKey) => {
 };
 
 function StripeApp({ stripePublishableKey }) {
+  const options = {
+    mode: 'payment',
+    currency: 'eur',
+    amount: 1337,
+	fields: {
+		billingDetails: 'never',
+	}
```
Comment (treoden) [inline packages/evershop/src/modules/stripe/pages/frontStore/checkout/Stripe.jsx:27]: Hi @ultimate-tester ,

Whoops. We can not hardcoded the amount and currency like this. It must be from the cart

Comment (ultimate-tester) [reply packages/evershop/src/modules/stripe/pages/frontStore/checkout/Stripe.jsx:27]: Would you be able to help me with this? My experience with React is limited so I don't have a clue on how to add these fields into context :(

Comment (treoden) [reply packages/evershop/src/modules/stripe/pages/frontStore/checkout/Stripe.jsx:27]: Sure, I will take care of it. Please give me sometime

**Comment by ultimate-tester** [comment]
I see this PR is merged but I cannot see that the pending task was resolved (I'm on my phone at the moment). Did you double check those hardcoded variables for e.g. amount?

**Comment by treoden** [comment]
Hi @ultimate-tester , 

I merged your PR to a feature branch and will continue fixing the issue there. 
Thanks for asking. I will let you know once I completed this 

Thanks

**Pull request merged**